hello/platforms/apple/Hello/.gitignore

```diff
@@ -5,5 +5,3 @@
 xcuserdata/
 DerivedData/
 .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
-
-Sources/HelloFFI.xcframework
```

Comment:
Why we need to track a binary into source code? 

We can always download it from GH actions.

Reply:
As mentioned in issue, when one uses the makefile and run `make apple` to build the framework and push it as a repo to be import as a swift package, including xcframework in .gitignore will fail the import.

Since this is an example, and I guess it will be easier for users to just push the built swift package to their repo without the extra step.

Reply:
This looks like a Swift Package Manager issue.

For this demo project, removing xcframework from .gitignore may not be a good idea.

Feel free to build other demos for SPM specifically.